src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -149,7 +155,7 @@ async def get_feed_details(self, require_success: bool = False) -> Any:
         tags_with_feed_ids = {
             tag: feed_id
             for tag, feed_id in current_feeds.items()
-            if (isinstance(tag, tuple) and len(current_feeds[tag]) > 0)
+            if (not isinstance(tag, tuple) and len(current_feeds[tag]) > 0)
```

Comment:
I don't think this fix works for #364 . You're now ensuring that tag is not a tuple, which seems correct, since you can't use a tuple as the key to a dictionary.

But in the second condition you're still getting the `len()` of a `current_feeds` dict value (a feed id, which is an `int`, no?), which is where the error came from. 

Another thing: you don't need to use `tag` in the second condition as a key, since you already have `feed_id` available after iterating over current feeds items. 

Is the second condition supposed to check if the `feed_id` is greater than zero?

Reply:
- not sure if I understand what you mean but I think a tuple can be a key in a dictionary.   
- The return values here are so that if the tag variable is a tuple then the value won't be needed , otherwise if the tag variable isn't a tuple then the value is needed and if the length of value is zero then disregard that item.
example response before filtering:
```{trb-usd-legacy: (b'\x04\xe3m.\x93\x17r\xe7\xfe\xb9m\xe8\x05\x16\xaa\x00\xa25\x1c\xfeI\xca\x17\xd8\xbd\x07D-\x1e\xb3\x9b9',), ('trb-usd-legacy', 'current_time'): 0, ('trb-usd-legacy', 'three_mos_ago'): 0,eth-jpy-legacy: (),'eth-jpy-legacy', 'current_time'): 0, ('eth-jpy-legacy', 'three_mos_ago'): 0}```
I'm trying to filter out the key that isn't a tuple that has a value which is tuple of length gt zero.

Reply:
cleared up on call, add comment about thhat data structure, then we're good to merge